lib/extendOrdWithCustom.js

```diff
@@ -49,21 +48,14 @@ function compareAndHandleCustomORDContentWithExistingContent(ordContent, customO
 }
 
 function getCustomORDContent(appConfig) {
-    let customORDContent;
-
-    if (appConfig.env?.customOrdContentFile) {
-        customORDContent = require(path.join(cds.root, appConfig.env.customOrdContentFile));
-    }
-    return customORDContent;
+    if (!appConfig.env?.customOrdContentFile) return;
+    const pathToCustomORDContent = path.join(cds.root, appConfig.env?.customOrdContentFile);
+    if (fs.existsSync(pathToCustomORDContent))  return require(pathToCustomORDContent);
```

Comment:
I would like have error log message here, including file not found and the path

Reply:
Done.